TwitchPlaysAssembly/Src/TwitchGame.cs

```diff
@@ -24,7 +24,9 @@ public class TwitchGame : MonoBehaviour
 	public Dictionary<string, Dictionary<string, double>> LastClaimedModule = new Dictionary<string, Dictionary<string, double>>();
 	public readonly List<CommandQueueItem> CommandQueue = new List<CommandQueueItem>();
 	public int callsNeeded = 1;
+	public int FindClaimUse = 0;
 	public List<string> CallingPlayers = new List<string>();
+	public List<string> FindClaimPlayers = new List<string>();
```

Comment:
This is just a personal thing, but I feel like this might be better implemented using a `Dictionary<string, int>` with the player's name being a the key and the number of claims they've allow to do being the value. The coroutine then just increase the value for every player until it's back to the `FindClaimLimit`. This would make it so we don't have an ever expanding list and removes the need for the `FindClaimUse` variable. You don't have to implement it this way, I might just do that before I merge it in.

Reply:
This would require the `Dictionary<string, int>` to add all of the players on the bomb. `FindClaimUse` is used so that any player can `!findclaim` a set number of modules. If you implemented it this way, a player who uses the command halfway through the bomb would only be able to use the value of `FindClaimLimit`'s number, while if you keep `FindClaimUse`, they get the same number as everyone else has had over the course of the bomb (effectively saving uses). I do like `Dictionary<string, int>`, so I will add that in my next commit.